Module-generation/frontmatter.py

```diff
@@ -14,89 +11,63 @@
+            ImageFile = open(DST+moduleName+"/logo.jpg", "wb+")    
+            ImageFile.write(readImage)
+            ImageFile.close()
+            sourceImage.close()
+
+        Tags=[]
         if "isGameplay" in parseData:
-            gameplay=parseData['isGameplay']
-        else:
-            gameplay=False
+            if (parseData['isGameplay']):
+                Tags.append("Gameplay")
 
         if "isServerSideOnly" in parseData:
-            serverside=parseData['isServerSideOnly']
-        else:
-            serverside=False
+            if(parseData['isServerSideOnly']):
+                Tags.append("Server")
 
         if "isWorld" in parseData:
-            world=parseData['isWorld']
-        else:
-            world=False
+            if (parseData['isWorld']):
+                Tags.append("World")
 
         if "isAugmentation" in parseData:
-            augment=parseData['isAugmentation']
-        else:
-            augment=False
+            if (parseData['isAugmentation']):
+                Tags.append("Augment")
         
         if "isLibrary" in parseData:
-            library=parseData['isLibrary']
-        else:
-            library=False
+            if(parseData['isLibrary']):
+                Tags.append("Library")
+       
         if "isAsset" in parseData:
-            asset=parseData['isAsset']
-        else:
-            asset=False
+            if(parseData['isAsset']):
+                Tags.append("Asset")
 
         if "isSpecific" in parseData:
-            specific=parseData['isSpecific']
-        else:
-            specific=False
-
-
-        moduleLogo="./logo.jpg"
-        moduleReadme=DIR+folders+"/README.md"
-
-        os.mkdir(DST+moduleName)
-        IndexMd=open(DST+moduleName+"/index.md","a+")
-        IndexMd.write('---\n')
-        IndexMd.write('posttype: module \n')
-        IndexMd.write('title: '+moduleName+'\n')
-        IndexMd.write('description: '+moduleDescription+'\n')
+            if(parseData['isSpecific']):
+                Tags.append("Specific")
 
-        if(os.path.isfile(moduleFile)):
-            IndexMd.write('cover: '+moduleLogo+'\n')
-        
-        Tags=[]
-        if(serverside):
-            Tags.append("Server")
-        if (gameplay):
-            Tags.append("Gameplay")
-        if (world):
-            Tags.append("World")
-        if (augment):
-            Tags.append("Augment")
-        if (library):
-            Tags.append("Library")
-        if (asset):
-            Tags.append("Asset")
-        if (specific):
-            Tags.append("Specific")
-        
         IndexMd.write("tags: "+"["+','.join(f'"{w}"' for w in Tags)+"]\n")
-        
         IndexMd.write('---\n')
 
         if(os.path.isfile(moduleReadme)):
             readmedata=open(moduleReadme,"r")
             IndexMd.write(readmedata.read())
         else:
-            IndexMd.write("No info available")
-
-
-
-
-
+            IndexMd.write("No info available")
```

Comment:
maybe we can find a better default for this. could be something along the lines of "XYZ is a (could add the first tag here or so) module for Terasology."

Reply:
Can you provide a text of 3 to 4 lines so that our cards description don't look empty